terraform/azure/main.tf

```
@@ -41,3 +55,105 @@ resource "azurerm_storage_account" "orsoarpadev" {
     environment = "dev"
   }
 }
+
+resource "azurerm_app_service_plan" "orsoarpadev" {
+  name                = module.app_service_plan_name.result
+  location            = azurerm_resource_group.orsoarpadev.location
+  resource_group_name = azurerm_resource_group.orsoarpadev.name
+
+  sku {
+    tier = "Free"
+    size = "F1"
+  }
+
+  tags = {
+    environment = "dev"
+  }
+}
+
+resource "azurerm_app_service" "orsoarpadev" {
+  name                = module.app_service_name.result
+  location            = azurerm_resource_group.orsoarpadev.location
+  resource_group_name = azurerm_resource_group.orsoarpadev.name
+  app_service_plan_id = azurerm_app_service_plan.orsoarpadev.id
+  https_only = true
+
+  site_config {
+    dotnet_framework_version = "v5.0"
+    scm_type                 = "GitHub"
+    http2_enabled = true
+    health_check_path = "/health"
+  }
+
+  app_settings = {
+    "Logging:IncludeScopes" = "false"
+    "Logging:LogLevel:Default" = "Trace"
+    "Logging:LogLevel:Microsoft" = "Warning"
+    "Logging:LogLevel:Microsoft.Hosting.Lifetime" = "Information"
+    "Logging:LogLevel:Microsoft.EntityFrameworkCore.Database.Command" = "Information"
+    "EmailConfiguration:From" = "dev@arpa.orso.co"
+    "EmailConfiguration:SmtpServer" = "localhost"
+    "EmailConfiguration:Port" = "25"
+    "EmailConfiguration:Username" = ""
+    "EmailConfiguration:Password" = ""
+    "EmailConfiguration:DefaultSubject" = "Message from ARPA"
+    "JwtConfiguration:TokenKey" = ""
+    "JwtConfiguration:Issuer" = "https://localhost:5001"
+    "JwtConfiguration:Audience" = "https://localhost:5001"
+    "JwtConfiguration:AccessTokenExpiryInMinutes" = "10"
+    "JwtConfiguration:RefreshTokenExpiryInDays" = "3"
+    "IdentityConfiguration:LockoutExpiryInMinutes" = "10"
+    "IdentityConfiguration:MaxFailedLoginAttempts" = "3"
+    "IdentityConfiguration:EmailConfirmationTokenExpiryInDays" = "3"
+    "IdentityConfiguration:DataProtectionTokenExpiryInHours" = "2"
+    "CorsConfiguration:AllowedOrigins:0" = "http://localhost:4200" // ToDo: Add storage url
```

How to get the connection strings / urls / instrumentation keys / sas tokens from the resources created with this script?

---

Is what you are looking for?
```suggestion
    "CorsConfiguration:AllowedOrigins:0" = azurerm_storage_account.orsoarpadev.primary_blob_endpoint 
```

---

exactly :) thanks!